ci/e2e/cleanup.yml

```diff
@@ -0,0 +1,27 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: kars7e/photon-k8s-ci
+    tag: v0.0.6
+
+# dispatch must be dispatch git repo.
+# cluster must contain file 'metadata' with JSON dict with two keys:
+#   "nodeIP": ingress controller IP, and "k8sURL": Kubernetes API URL
+# dispatch-cli must contain "dispatch" binary
+inputs:
+- name: dispatch
+- name: cluster
+- name: dispatch-cli
+
+run:
+  path: /bin/bash
+  args:
+  - -c
+  - |
+    set -e -x -u
+    source dispatch/ci/e2e/config-k8s-env.sh
+    helm delete --purge dispatch api-gateway openfaas ingress || true
```

Comment:
we could also use `dispatch uninstall`

Reply:
I was using it originally but needed more control what is deleted during testing. Also, verbosity would be useful but debug prints config content which contains sensitive credentials. We can change it once the pipeline gets stable.